thesis-de/content-de.tex

```diff
@@ -1337,6 +1337,32 @@ \subsubsection{Durchführung Evaluierung Frontend}
   \1 Kann keine zusätzliche Datenbank ansprechen, somit ist eine Datensicherung
   nicht gewährleistet  
 \end{outline}
+Nach Abschluss der Evaluierung der einzelnen Softwarelösungen für das Web
+Frontend, sind folgende drei Weboberflächen in die nähere Auswahl genommen
+worden:
+
+\begin{outline}
+  \1 Grafana
+  \1 Graphite
+  \2 Zabbix-Frontend
+\end{outline}
+
+Alle drei Softwarelösungen erfüllen, die bei Beginn des Projektes definierten
+Anforderungskriterien, weshalb die Produkte in die nähere Anlyse aufgenommen
+wurden. Hier wurde zunächst geschaut, ob ein System eine bestimmte
+Besonderheiten oder Funktion aufweist. Dabei ist aufgefallen, dass das 
+Zabbix-Frontend ohne einen Zabbix Server nicht verwendet werden kann. Der
+Zabbix Server ist eins von Zabbix eigen Entwickelter Server, der für die 
```

Comment:
"ist eins von Zabbix eigen Entwickelter Server" -> ist ein von Zabbix LLC entwickelter Netzwerkdienst. Er empfängt Metriken von Agents, bereitet diese auf und speichert sie anschliesend in einer Datenbank.